### ISSUE TITLE
feature request: play youtube and instagram videos?

### ISSUE THREAD

**Issue by jritz611** [extesy/hoverzoom#49]

**Comment by extesy** [comment]
@jritz611 Youtube previews are already implemented. Where can I find a page with a bunch of instragram video links?

**Comment by jritz611** [comment]
NBA Reddit has some. YouTube doesn't work for me at least in Reddit.

**Comment by jritz611** [comment]
https://www.reddit.com/r/nba/comments/499hma/kobe_bryant_no_look_randle_finishes/ <https://www.reddit.com/r/nba/comments/499hma/kobe_bryant_no_look_randle_finishes/>

**Comment by Fastjur** [comment]
Hmm, for me it does actually load YouTube video's, in fact, I want to disable it. But putting it on the block list (youtube.com, youtu.be) doesn't work. It keeps on loading them.

**Comment by extesy** [comment]
@Fastjur You can turn off "zoom videos" link in hoverzoom+ options.

**Comment by Fastjur** [comment]
Unfortunately this also disables the .webm support (which I actually want). I'll just have to deal with the youtube links I guess.

Maybe I'm adding the links in wrong?
Should I use wildcards like this: `*youtube.com*` and `*youtu.be*`

**Comment by extesy** [comment]
Blacklist/whitelist is for sites which have outgoing links, like reddit.com. It's not for the link destination. So adding youtube to blacklist would disable hoverzoom on youtube.com.

**Comment by extesy** [comment]
By the way, what's wrong with previewing youtube videos? It's uses the same webm stream as for imgur.com videos.

**Comment by Fastjur** [comment]
@extesy Aha, I misinterpreted the blacklist then. For me it is annoying because a lot of times when I move my cursor to another screen and it hits a youtube video on reddit, it will start playing that, even though my mouse is on another screen. Most video's have music or a lot of sound in it

**Comment by extesy** [comment]
@Fastjur You can increase the video preview delay a little bit which would help with that. You can also mute videos, or set custom volume (in the next release).

**Comment by danielsamuels** [comment]
I'd much rather see video hosting websites (YouTube, Vimeo etc) on a different setting to "videos" such as gifvs. One type will never have sound, one typically will. Even when setting it to mute by default, I'm still not going to want to watch the (typically) longer length video while keeping my cursor in the same place.

**Comment by Fastjur** [comment]
@danielsamuels Yes I agree, but .gifv is considered as a video though. I'd like to see a way of blocking certain domains from showing up in hoverzoom (thus, if I block youtube.com) it will not zoom in on any youtube video's on any page

**Comment by extesy** [comment]
@Fastjur It looks like what you want is the ability to disable some hz plugins. There are separate plugins for each image/video hosting domain, so disabling youtube plugin will disable previewing videos from youtube.com domain.

**Comment by Fastjur** [comment]
Hi @extesy,

sorry for my possbile ignorance in this. Is that already possible? If so how. If this is not possible, then that is indeed exactly what I'd like to see!

**Comment by extesy** [comment]
It's not possible yet, this functionality needs to be added.

**Comment by Fastjur** [comment]
Alright, shall I open an issue (feature request) for this?

**Comment by extesy** [comment]
Sure, go ahead
